com.unity.ml-agents/Runtime/Utilities.cs

```diff
@@ -75,83 +74,6 @@ public static int[] CumSum(int[] input)
             return result;
         }
 
-        /// <summary>
-        /// Shifts list elements to the left by the specified amount (in-place).
-        /// <param name="list">
-        /// List whose elements will be shifted
-        /// </param>
-        /// <param name="shiftAmount">
-        /// Amount to shift the elements to the left by
-        /// </param>
-        /// </summary>
-        public static void ShiftLeft<T>(List<T> list, int shiftAmount)
```

Comment:
We did not have any tests for these?

Reply:
Nope 
![image](https://user-images.githubusercontent.com/6877802/75298327-03bcfb00-57e7-11ea-9b04-9a8a43b957c5.png)